plotting/plotter_3d.py

```
@@ -0,0 +1,375 @@
+
+                b = [x for x in data if x.get('key') == a]
+                a = b[0]
+                self.area[idx] = a
+            else:
+                self.points = copy.deepcopy(np.array(a['polygons']))
+                a['polygons'] = self.points.tolist()
+                a['name'] = " "
+
+            rings = [LinearRing(po) for po in a['polygons']]
+            if len(rings) > 1:
+                u = cascaded_union(rings)
```

Ruff (0.12.2)

Undefined name `cascaded_union`

(F821)